### ISSUE TITLE
Remove `passive` event listener flag? 

### ISSUE THREAD

**Issue by trotzig** [brigade/react-waypoint#205]
I was reading up on [`EventTarget.addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Improving_scrolling_performance_with_passive_listeners) and came across this section: 

> Setting passive isn't important for the basic scroll event, as it cannot be canceled, so its listener can't block page rendering anyway.

We introduced passive listeners in 3ad940e, but I think that has no effect on performance and we can remove it. We do use it for both the `scroll` event and the `resize` event. I think it might have effect for `resize`, but performance isn't really important for that event (people don't normally resize their browsers).

@lencioni I'm interested to get your input here.

**Comment by jmeas** [comment]
After reading that article, and a few other ones on the topic, this _seems_ right to me. But I'll also defer to @lencioni .

**Comment by lencioni** [comment]
Yeah, I think you are right--scroll and resize events can't be canceled. It seems fine to remove it.
